tests/api_test.py

```diff
@@ -1413,6 +1413,19 @@ def f(x):
       return np.zeros((2,))
     f(np.zeros((5,)))
 
+  def test_xla_constant_dedup(self):
+    y = np.array([7, 14], dtype=np.float32)
+    def f(x):
+      return x + y + y
+
+    x = np.array([1, 2], dtype=np.float32)
+    hlo_lines = jax.xla_computation(f)(x).as_hlo_text().split('\n')
+    hlo_lines = set([s.strip() for s in hlo_lines])
+    hlo_y1 = 'constant.1 = f32[2]{0} constant({7, 14})'
+    hlo_y2 = 'constant.2 = f32[2]{0} constant({7, 14})'
+    assert hlo_y1 in hlo_lines
```

Comment:
```suggestion
    self.assertIn('constant.1 = f32[2]{0} constant({7, 14})', hlo_lines)
```

Using `assertIn` prints more info to the log upon failure.